src/schema/ecommerce/types/order_line_item.js

```diff
@@ -27,6 +27,10 @@ export const OrderLineItemType = new GraphQLObjectType({
       description: "Edition set on the artwork",
       resolve: ({ editionSetId }) => editionSetId,
     },
+    priceCents: {
+      type: GraphQLInt,
+      description: "Price in cents",
+    },
```

Comment:
Unclear if this is the price of 1 item or the subtotal of `quantity` items. The description could clarify.